group1/comm/static/comm_tool.js

```diff
@@ -490,25 +707,46 @@ function populate_user_list() {
     $.getJSON('http://' + server_host + ':3000/users', function(connected_users){
 
       // make sure the current user is included in the list
-      connected_users.push(user);
+      //connected_users.push(user);
       online_users = _.unique(connected_users);
 
+
       global_user_list = all_users;
       all_users.forEach(function(user) {
+          add_new_indvRoom(user.id);
         var user_link = $('<li />', {
-          'class': _.contains(online_users, user.username) ? 'user' : 'user disabled',
+          'class': _.contains(online_users, user.id) ? 'user' : 'user',
+          'id':user.id,
           'html':
           $('<a />', {
-            'href': '#'
+            'href': '#',
+              'class': 'list-user-item user-link'
+
           })
-        .append(user.username)
+        .append(user.username).append($('<span />',{
+        'class': 'badge'
+        }))
         });
 
         $('ul.user_list').append(user_link);
+
       });
     });
+
   });
 }
+function add_new_indvRoom(userId){
+    $.getJSON('http://localhost:8000/api/indvrooms/?format=json', function (data) {
```

Comment:
Any references to the server in comm_tool.js should be of the format:
'http://' + server_host + ':' + server_port + 'API PATH'

for example: 
'http://' + server_host + ':' + server_port + '/api/indvrooms/?format=json'

References to localhost in comm_tool.js will not resolve correctly, as they point to your local machine and not to the server where the files are actually hosted.